javascript/javascript1/week2/flight-booking.js

```diff
@@ -0,0 +1,15 @@
+function getFullName(firstname, surname, useFormalName, gender) {
+  if (useFormalName === true && gender === "male") {
```

Comment:
it is good , nice and clean .You could alternatively write `if (useFormalName)`, it is the same. also in line 4.